mount-tos-image/README.md

```diff
@@ -47,6 +50,12 @@ But since all calculations in the FAT are done using clusters all we need to do
 
 Of course this manipulation is not done directly on the SD-/CF-Card. Instead, the script makes heavy use of some nice and cool Linux features (FUSE and device mapper).
 
+## How to start the script
+
+* Connect the TOS medium of your choice as block device to the linux system or simply use an image file  
+* Start the script from within a working directory on a file system that is capable of creating spares files (e.g. btrfs, ext4, ...)
```

Comment:
Please fix the typo: spares => sparse

Reply:
Thank you very much! I fixed the typo and corrected the indention.